apps/source_academy/lib/source_academy/user.ex

```diff
@@ -0,0 +1,48 @@
+defmodule SourceAcademy.User do
+  @moduledoc """
+    The User entity contains basic user data such as name and e-mail.
+    A user can have many authorisations from different providers.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias SourceAcademy.Repo
+
+  @type t :: %__MODULE__{}
+
+  schema "users" do
+    field :name, :string
```

Comment:
I think we should split to first_name and last_name for now, make last name optional